hazel.bzl

```diff
@@ -148,6 +148,7 @@ def hazel_custom_package_github(
     github_user,
     github_repo,
     repo_sha,
```

Comment:
Can we call this attribute `strip_prefix` or something similar to be more like other rules?